spec.bs

```diff
@@ -975,23 +976,36 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
         1. [=map/Set=] |auctionConfig|'s [=auction config/per buyer signals=][|buyer|] to
           |signalsString|.
     * To handle an error, set |auctionConfig|'s [=auction config/per buyer signals=] to failure.
-1. If |config|["{{AuctionAdConfig/perBuyerTimeouts}}"] [=map/exists=]:
-  1. Set |auctionConfig|'s [=auction config/per buyer timeouts=] to
-      |config|["{{AuctionAdConfig/perBuyerTimeouts}}"].
-  1. [=Handle an input promise in configuration=] given |auctionConfig| and |auctionConfig|'s
-    [=auction config/per buyer timeouts=]:
-    * To parse the value |result|:
-      1. Set |auctionConfig|'s [=auction config/per buyer timeouts=] to a new [=ordered map=] whose
-        [=map/keys=] are [=origins=] and whose [=map/values=] are [=durations=] in milliseconds.
-      1. [=map/For each=] |key| → |value| of |result|:
-        1. If |key| is "*", then set |auctionConfig|'s [=auction config/all buyers timeout=]
-          to |value| in milliseconds or 500 milliseconds, whichever is smaller, and
-          [=iteration/continue=].
-        1. Let |buyer| be the result of [=parsing an https origin=] with |key|. If |buyer| is
-          failure, [=exception/throw=] a {{TypeError}}.
-        1. [=map/Set=] |auctionConfig|'s [=auction config/per buyer timeouts=][|buyer|] to
-          |value| in milliseconds or 500 milliseconds, whichever is smaller.
-    * To handle an error, set |auctionConfig|'s [=auction config/per buyer timeouts=] to failure.
+1. For each |configMember|, |auctionConfigField|, |auctionConfigFieldAll| in the following table

```

Comment:
For such a small table of options, this kind of meta loop is a little hard to read, so I'd slightly prefer to just be really explicit so that all of the `Set |auctionConfig|'s to |auctionConfigField|` stuff can be a lot more readable and linkable. It doesn't _really_ matter, but in general this spec is really hard to read, so making it incrementally more straightforward would probably be a good thing. But this is optional, totally up to you.

Alternatively, maybe we can rename these to something like `auctionConfigPerBuyerTimeout` and `auctionConfigAllBuyer[...]` (or maybe I'm misunderstanding!)